core/keyserver/keyserver.go

```diff
@@ -273,6 +273,12 @@ func (s *Server) ListEntryHistory(ctx context.Context, in *pb.ListEntryHistoryRe
 	}, nil
 }
 
+// ListUserRevisions returns a list of revisions covering a period of time.
+func (s *Server) ListUserRevisions(ctx context.Context, in *pb.ListUserRevisionsRequest) (
```

Comment:
the multi-lining here seems weird. Could you move the opening parens down to the line below?

Reply:
If you don't leave the hanging open parens, Go thinks you forgot the `{` at the start of the function definition (doesn't compile).